schema/show.js

```diff
@@ -315,7 +315,7 @@ const Show = {
       .then(show => {
         if (!show.displayable && !show.is_reference) return new Error('Show Not Found');
         return show;
-      });
+      }).catch(() => null);
```

Comment:
Aha, gotcha 👍 

Would it possible to ensure this old truly swallows _those_ specific errors you are thinking of? I’m slightly worried that this could mask any other error that we do want to know about.